apps/web/src/components/templates/in-app-editor/TemplateInAppEditor.tsx

```diff
@@ -33,15 +33,6 @@ export function TemplateInAppEditor({ control, index }: { control: Control<IForm
     },
   });
 
-  const { mutateAsync: deleteFeedById } = useMutation<
```

Comment:
Is the removal here intentional?

Reply:
Yes, it also exists in another componenet. This one was not used. @scopsy